src/effects/effectsmanager.cpp

```diff
@@ -347,63 +349,65 @@ void EffectsManager::setup() {
     addStandardEffectRack();
     addOutputsEffectRack();
 
-    EffectChainPointer pChain(new EffectChain(
-           this, "org.mixxx.effectchain.flanger"));
-    pChain->setName(tr("Flanger"));
-    EffectPointer pEffect = instantiateEffect(
-           "org.mixxx.effects.flanger");
-    pChain->addEffect(pEffect);
-    m_pEffectChainManager->addEffectChain(pChain);
-
-    pChain = EffectChainPointer(new EffectChain(
-            this, "org.mixxx.effectchain.bitcrusher"));
-    pChain->setName(tr("BitCrusher"));
-    pEffect = instantiateEffect("org.mixxx.effects.bitcrusher");
-    pChain->addEffect(pEffect);
-    m_pEffectChainManager->addEffectChain(pChain);
-
-    pChain = EffectChainPointer(new EffectChain(
-            this, "org.mixxx.effectchain.filter"));
-    pChain->setName(tr("Filter"));
-    pEffect = instantiateEffect("org.mixxx.effects.filter");
-    pChain->addEffect(pEffect);
-    m_pEffectChainManager->addEffectChain(pChain);
-
-#ifndef __MACAPPSTORE__
-    pChain = EffectChainPointer(new EffectChain(
-            this, "org.mixxx.effectchain.reverb"));
-    pChain->setName(tr("Reverb"));
-    pEffect = instantiateEffect("org.mixxx.effects.reverb");
-    pChain->addEffect(pEffect);
-    m_pEffectChainManager->addEffectChain(pChain);
-#endif
-
-    pChain = EffectChainPointer(new EffectChain(
-            this, "org.mixxx.effectchain.echo"));
-    pChain->setName(tr("Echo"));
-    pEffect = instantiateEffect("org.mixxx.effects.echo");
-    pChain->addEffect(pEffect);
-    m_pEffectChainManager->addEffectChain(pChain);
-
-    pChain = EffectChainPointer(new EffectChain(
-            this, "org.mixxx.effectchain.autopan"));
-    pChain->setName(tr("AutoPan"));
-    pEffect = instantiateEffect("org.mixxx.effects.autopan");
-    pChain->addEffect(pEffect);
-    m_pEffectChainManager->addEffectChain(pChain);
-
-    pChain = EffectChainPointer(new EffectChain(
-            this, "org.mixxx.effectchain.tremolo"));
-    pChain->setName(tr("Tremolo"));
-    pEffect = instantiateEffect("org.mixxx.effects.tremolo");
-    pChain->addEffect(pEffect);
-    m_pEffectChainManager->addEffectChain(pChain);
-}
-
-void EffectsManager::loadEffectChains() {
-    // populate rack and restore state from effects.xml
-    m_pEffectChainManager->loadEffectChains();
-}
+    // NOTE(Kshitij) : Why are we adding single-effect effect chains on startup?
```

Comment:
This was a really ugly hack around some of the skins in the initial implementation of the effects system in Mixxx 2.0 not showing chains at all. Remove this outdated code please.